services/braintree-webhook.php

```diff
@@ -504,6 +504,8 @@
 	pmpro_braintreeWebhookExit();
 }
 
+do_action( 'pmpro_webhook_unhandled' );
```

Comment:
Let's add hook doc info via phpdoc for new actions.

```suggestion
/**
 * Allow hooking into after a webhook has been run but was not handled.
 *
 * @since TBD
 *
 * @param string $gateway The gateway the webhook was not handled for.
 */
do_action( 'pmpro_webhook_unhandled', PMPRO_DOING_WEBHOOK );
```